### PR TITLE
bridge more channels

### PR DESCRIPTION
**Pull request by mplscorwin**
Get new/renamed channels bridged.

### CODE REVIEW

**Comment by dmehus** [comment]
@mplscorwin Are you wanting to add MirahezeBot to the channels listed in 4ba668e? I just want to make sure I'm understanding the commit correctly. If //so//, sounds good to me. :)

File: modules/profile/templates/fhconfig.json [hunk 1]
```diff
@@ -5,8 +5,14 @@
       "discordToken":"<%= @discordfh_token %>",
       "ircNickColor":false,
       "channelMapping":{
+         ;; public
```
Comment (MacFan4000) [inline modules/profile/templates/fhconfig.json:8]: Json doesn’t support comments

File: modules/profile/templates/fhconfig.json [hunk 2]
```diff
@@ -5,8 +5,14 @@
...
          "#fosshost":"#fosshost",
+         "#fosshost-incident":"#fosshost-incident",
+         "#fosshost-social":"#fosshost-social",
+         ;; private
```
Comment (MacFan4000) [inline modules/profile/templates/fhconfig.json:12]: See above

File: modules/profile/templates/fhconfig.json [hunk 3]
```diff
@@ -5,8 +5,14 @@
...
          "#fosshost-volunteers":"#fosshost-volunteers",
+         "#fosshost-exec":"#fosshost-exec",
+         ;; bots only ;)
```
Comment (MacFan4000) [inline modules/profile/templates/fhconfig.json:15]: See above

**Review by MacFan4000 — DISMISSED** [review]

**Comment by RhinosF1** [comment]
> @mplscorwin Are you wanting to add MirahezeBot to the channels listed in [4ba668e](https://github.com/MirahezeBots/puppet/commit/4ba668e3dcd4d24957a6ef228a9ba98b6d008e9d)? I just want to make sure I'm understanding the commit correctly. If //so//, sounds good to me. :)

No the relay

**Review by RhinosF1 — APPROVED** [review]
Should be fine

**Pull request merged**